DigitalLearningSolutions.Web/Views/Frameworks/Developer/AddCompetencyLearningResources.cshtml

```diff
@@ -26,21 +26,45 @@
     @Model.NameOfCompetency
   </p>
 </div>
-<h2>Search the Learning Hub</h2>
-<div class="nhsuk-grid-row">
-  <div class="nhsuk-grid-column-three-quarters">
-    <form method="get" role="search">
+<form method="post" role="search" asp-controller="Frameworks" asp-action="SearchLearningResources">
+  <h2>Search the Learning Hub</h2>
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-three-quarters">
       <div class="search-box-container" id="search">
-        <label class="nhsuk-u-visually-hidden" for="search-field">Search</label>
-        <input class="search-box" id="search-field" name="searchString" type="search" placeholder="Search" autocomplete="off">
-        <button class="nhsuk-search__submit nhsuk-search__submit sort-submit" type="submit">
+        @Html.LabelFor(m => m.SearchText, "Search", new { @class = "nhsuk-u-visually-hidden", @for = "search-field" })
+        @Html.TextBoxFor(m => m.SearchText, new { @class = "search-box", id = "search-field", name = "searchString", type = "search", placeholder = "Search", autocomplete = "off" })
+        <button class="nhsuk-search__submit sort-submit" type="submit">
```

Comment:
This button should be converted to a link to call a Get method and should have search string and page number as route values, e.g.:
```
asp-route-search="@Model.SearchString"
asp-route-page="1"
```

Reply:
I have changed the `form` http method to `GET ` and passed page number as a hidden field. That way I don't need to update the link as user changes search text, no need to amend styles for the button, etc.